### PR TITLE
Unit test for survey(eval framework) added

### PR DESCRIPTION
**Pull request by SarahhhC**
Added unit tests for survey. Please have a look!

@ethkim @nlpenguin @lyw0149 @heartjigi @HewonJeong

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "express": "^4.14.0",
     "express-session": "^1.14.1",
     "gulp": "^3.9.1",
+    "gulp-istanbul": "^1.1.1",
```
Comment (lyw0149) [inline package.json]: Istanbul is in both normal dependencies and dev-dependencies. is it right??

Comment (SarahhhC) [inline package.json]: That's true. will remove one in normal.

File: src/test/controllers/survey.controller.test.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        uri: API_BASE_URL + '/create',
+        form: mockSurveyData_mentee.surveyA001_1,
+        resolveWithFullResponse: true,
+        json: true,
+      };
+
+      rp(options)
+        .then((result) => {
+          result.statusCode.should.equal(200);
+          result.body.survey_id.should.equal(mockSurveyData_mentee.surveyA001_1.survey_id);
```
Comment (ethkim) [inline src/test/controllers/survey.controller.test.js]: indentation? linebreak?

Comment (ethkim) [inline src/test/controllers/survey.controller.test.js]: i forget what was the max # of chars per line. was it 100?

File: src/test/fixtures/surveyA001_1.js [hunk 3]
```diff
@@ -0,0 +1,227 @@
+export default {
+  surveyA001_1: {
+    survey_id: 'A001-1',
+    questions: [
+      {
+        question_index: 0,
+        question: '왜 해당 멘토를 선택하였는가?',
```
Comment (ethkim) [inline src/test/fixtures/surveyA001_1.js]: Set the question to English (given that the answers are also in English)?

File: src/test/controllers/survey.controller.test.js (hunk 2 above)
Comment (SarahhhC) [inline src/test/controllers/survey.controller.test.js]: Yes, it need to be shorten. will change mock data as
`mockSurveyData_mentee => menteeData`
`mockSurveyData_mentee => mentorData`
like this!

File: src/test/fixtures/surveyA001_1.js (hunk 3 above)
Comment (SarahhhC) [inline src/test/fixtures/surveyA001_1.js]: I set this from 'Yoda Sprint Planning' sprint#4 and it wasn't English..T.T I updated it please check!

Comment (ochanje210) [inline src/test/fixtures/surveyA001_1.js]: English

Comment (SarahhhC) [inline src/test/fixtures/surveyA001_1.js]: Hmm...I think I misunderstood! Questions should be Korean.T.T

**Comment by lyw0149** [comment]
LGTM thanks!! 
@SarahhhC

**Pull request merged**